user_docs/en/userGuide.md

```diff
@@ -43,15 +43,15 @@ Major highlights include:
 
 * Operating Systems: all 32-bit and 64-bit editions of Windows 8.1, Windows 10, Windows 11, and all Server Operating Systems starting from Windows Server 2012 R2.
   * both AMD64 and ARM64 variants of Windows are supported.
-  * Note 32-bit operating systems are no longer under active support.
+  * Note that 32-bit operating systems are no longer under active support.
   * Note that Windows 8.1 and Windows Server versions older than 2022 are no longer under active support.
-* at least 150 MB of storage space.
+* at least 200 MB of storage space.
```

Comment:
Maybe it's worth considerably upping this? At the very least we need to make it installer size + install dir size. Also taking into consideration add-on store cache files, expanded user config, etc.

```suggestion
* at least 500 MB of storage space.
```

Reply:
and also means we don't have to update it as frequently as it grows over time